docs/advanced/naming.md

```diff
@@ -164,16 +202,19 @@ Anime.Name.2022.S01.E01-E11.1080p.BluRay.FLAC2.0.H.265-Group
 Anime.Name.2022.S01.E12-E23.1080p.BluRay.FLAC2.0.H.265-Group
 ```
 
-### Nyaa
+## Nyaa
 
-#### Title
+In general, a Nyaa release should include:
 
-- Same guidelines as Folder, simply add more aliases at the end for easier searching.
-- You can also put the group tag before everything else in the Nyaa title.
+- The same guidelines followed for [folders](#folders)
+- Additional details/aliases at the end of the title for better searching (e.g. alternate show titles, season number, cour number)
+- A raw link to [MediaInfo](https://mediaarea.net/en/MediaInfo) information
+  - For season releases, we recommend using the first episode
+- A link to a [quality comparison](/tutorials/comparison) including the video used in the release
```

Comment:
Additionally, you should cover what you did in the description (i.e "A track taken from B, used C to demux it, fixed audio clipping with adobe audition, retimed subs etc etc")